lib/MusicBrainz/Server/Controller/ArtistCredit.pm

```diff
@@ -79,9 +83,9 @@ map {
         );
     };
 
-    find_meta(__PACKAGE__)->add_method($_ => $method);
+    find_meta(__PACKAGE__)->add_method($entity_type => $method);
     find_meta(__PACKAGE__)->register_method_attributes($method, [q{Chained('load')}, "PathPart('$url')"]);
-} entities_with('artist_credits');
+};
```

Comment:
The semi-colon here is not needed

Reply:
I think later critic rules will catch any other that we missed here, but fixed this :)